cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -231,8 +231,10 @@ protected void configure() {
         setupClasspath();
 
         // Set local tmp dir to an absolute location in the twill runnable otherwise Hive complains
+        String localScratchPath = System.getProperty("java.io.tmpdir") + File.separator +
```

Comment:
Preserving the current behavior may not be the right thing. Can you check if runs in standalone will it pollute the "/tmp"? (for distributed mode, it's probably ok, since "java.io.tmpdir" is set to container local directory).